services/core-api/app/api/now_applications/response_models.py

```diff
@@ -586,8 +589,7 @@ def format(self, value):
         'security_received_date': Date,
         'security_not_required': fields.Boolean,
         'security_not_required_reason': fields.String,
-        'last_updated_date': Date,
-        'last_updated_by': fields.String,
+        'last_updated_date': Date
```

Comment:
The FE still uses these values, is this a template-specific response model?

Reply:
I'm assuming so

Reply:
Yup, it's the model that is used just for the NoW document.